src/mixins/filing-template-mixin.ts

```diff
@@ -656,7 +674,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
     }
 
     // store Document Optional Email
-    this.setDocumentOptionalEmail(filing.header.documentOptionalEmail)
+    this.setDocumentOptionalEmail(filing.header.documentOptionalEmail || '')
```

Comment:
This fallback (and others below) matches the initial values for these fields (in store state).

Without this fallback, it's possible the store state values could be undefined, which could break reactivity.